src/modules/asset-metrics/asset-metrics.module.ts

```diff
@@ -0,0 +1,17 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { AssetFundamentalsEntity } from 'lib-typeorm';
+import { AssetMetricsEntity } from 'lib-typeorm';
+import { AssetFundamentalsController } from '../asset-fundamentals/asset-fundamentals.controller';
+import { AssetFundamentalsService } from '../asset-fundamentals/asset-fundamentals.service';
+
+
+@Module({
+  imports: [
+    TypeOrmModule.forFeature([AssetFundamentalsEntity, AssetMetricsEntity]),
+  ],
+  providers: [AssetFundamentalsService],
+  controllers: [AssetFundamentalsController],
+  exports: [TypeOrmModule],
```

Comment:
why we are exporting typeorm module

Reply:
this was to be done in the library instead. have made the changes in the library